be/src/exec/vectorized/aggregate/aggregate_blocking_node.cpp

```diff
@@ -42,6 +42,11 @@ Status AggregateBlockingNode::open(RuntimeState* state) {
 
         DCHECK_LE(chunk->num_rows(), config::vector_chunk_size);
 
+        _aggregator->process_input_limit(&chunk);
```

Comment:
This implementation is wrong, we should limit the hash map size, not the input size.